web-app/css/smartR.css

```diff
@@ -185,18 +185,28 @@ div.sr-status-and-output-container:empty:before {
     color:red;
 }
 
-.mybutton {
+.sr-contextMenu {
```

Comment:
just coding style question. Shall we avoid camel case in css naming? dash is preferable e.g.:

.this-is-a-css-class {}

I guess for css especially when we are looking at the html will be more readable.